human_experiments/scripts/run_minecraft_metadata_validation.py

```diff
@@ -0,0 +1,92 @@
+import sys 
+import os
+import json
+import argparse
+import datetime
+import pandas as pd
+from termcolor import colored
+from time import sleep
+
+def check_time_difference(trial_start, trial_end):
+    #check difference in time
+    sleep(0.25)
+    trial_start = trial_start.split('T')[0] + " " + trial_start.split('T')[1].split('Z')[0]
+    trial_end = trial_end.split('T')[0] + " " + trial_end.split('T')[1].split('Z')[0]
+
+    print(colored('\n[Status] Mission started at time:', 'red', attrs=['bold']), trial_start)
+    print(colored('\n[Status] Mission ended at time:', 'red', attrs=['bold']), trial_end)
+
+    trial_start = datetime.datetime.strptime(str(trial_start), '%Y-%m-%d %H:%M:%S.%f')
+    trial_start = trial_start.timestamp()
+    trial_end = datetime.datetime.strptime(str(trial_end), '%Y-%m-%d %H:%M:%S.%f')
+    trial_end = trial_end.timestamp()
+    delta = int(float(trial_end) - float(trial_start))
+    min, sec = divmod(delta, 60)
+
+    if min > 0:
+        print(colored('\n[Status] Trial lasted for: ', 'red', attrs=['bold']), min, 'Minutes', sec, 'Seconds')
+    else:
+        print(colored('[Error] Timing for mission is off by a large margin','red'), u'\N{cross mark}')
+
+def read_subject_id(TrialMessages):
+    #Display subject IDs
+    sleep(0.25)
```

Comment:
Why do we have these 0.25s sleeps?

Reply:
> Why do we have these 0.25s sleeps?

Well, this gives the experimenter the time to read what is displayed on the terminal. I can remove it if you feel it's not needed.

Reply:
I don't think we need them - the experimenter can always just scroll up in the terminal (I'm assuming there is not a flood of things printed to stdout, so this should feasible. Also, the experimenter can always simply run the script again if they missed something, right?

Reply:
I will remove em in that case. Yes, the experimenter can simply run the script again if they want. Everything will show up on the terminal so no need to run it again. Screenshot -> https://github.com/ml4ai/tomcat/issues/385